### DISCUSSION TITLE
webhook for environment variable updating

### DISCUSSION THREAD

**Discussion by Geczy** [coollabsio/coolify#1591] [Feature Requests]
to integrate with secret managers

my use case is doppler https://docs.doppler.com/docs/webhooks#accepting-webhooks

**Comment by ErlandssonA** [comment]
I just started using Doppler and it would be amazing to be able to be able to integrate it with Coolify!

**Comment by silvyyuliana** [comment]
I recently started using Doppler and would like native integration with Coolify to make managing secrets easier and more secure.

**Comment by ARHAEEM** [comment]
Droppler should be part of Coolify, it’s a pretty way to unify secrets across platforms.

**Comment by ErlandssonA** [comment]
I ended up self-hosting Infisical and then moving all my apps etc to Dockerfiles. That way I can import the secrets automatically at buildtime and runtime. Only have to add the infisical tokens, projectid etc to the secrets and it works like a charm. It takes some time to set everything up properly, but works like a charm now and its really easy to manage everything.

**Comment by Wdestroier** [comment]
Apparently there are many alternatives: Doppler, Infisical, Keyshade...
